packages/react-liveness/src/components/FaceLivenessDetector/FaceLivenessDetector.tsx

```diff
@@ -16,8 +16,8 @@ export interface FaceLivenessDetectorProps
 }
 
 const credentialProvider = async () => {
-  const credentials = (await fetchAuthSession()) as AwsTemporaryCredentials;
-  return credentials;
+  const { credentials } = await fetchAuthSession();
+  return credentials as AwsTemporaryCredentials;
```

Comment:
do we still need our type or is fetchAuthSession strongly typed now?

Reply:
Hm actually think this is not needed now, updating